common/membership/rpMonitor.go

```diff
@@ -115,6 +115,10 @@ func (rpo *ringpopMonitor) WhoAmI() (*HostInfo, error) {
 	return NewHostInfo(address, labels.AsMap()), nil
 }
 
+func (rpo *ringpopMonitor) EvictSelf() error {
+	return rpo.rp.SelfEvict()
```

Comment:
maybe keep using name EvictSelf?

Reply:
the underlying ringpop library uses the name SelfEvict() which cannot be changed. Yes, we can use the same name. I renamed it because method names usually start with "verb" which makes them very readable. And this underlying library is only invoked in this place.. so nobody cares about this code.. if you feel strongly, I can use the same name for consistency.

Reply:
+1 for making method name start with verb.